script/helpers/PantosBaseAddresses.s.sol

```diff
@@ -40,8 +40,8 @@ abstract contract PantosBaseAddresses is PantosBaseScript {
         PAN_CELO,
         PAN_CRO,
         PAN_ETH,
-        PAN_FTM,
-        PAN_MATIC
+        PAN_POL,
+        PAN_S
```

Comment:
just curious why not use `PAN_SONIC`?

Reply:
Sonic is the name of the new network, their native coin is going to be called `S`

Reply:
very creative name :rocket: